### DISCUSSION TITLE
Can I use FireDBG on project built with rustc 1.75 (nightly)

### DISCUSSION THREAD

**Discussion by danzou1ge6** [SeaQL/FireDBG.for.Rust#22] [General]
FireDBG reports

```
thread 'main' panicked at debugger/src/debugger.rs:1129:9:
Mismatched `rustc` version: expect `1.74`, found `1.75`. Please install a matching FireDBG version.
```

**Comment by tyt2y3** [comment]
Due to ABI (in)stability, we need to build the binaries for every major rustc. You can try building from source, and we'd probably release an `rc` for 1.75 soon, if no major incompatibilities are found.

**Comment by tyt2y3** [comment]
You can try https://github.com/SeaQL/FireDBG.for.Rust/releases/tag/1.75.0-beta.1 now!

**Comment by danzou1ge6** [comment]
Thanks. I've got it running.
